{{cookiecutter.project_slug}}/backend/celeryworker.dockerfile

```diff
@@ -1,6 +1,7 @@
 FROM python:3.7
 
-RUN pip install raven celery~=4.3 passlib[bcrypt] tenacity requests "fastapi>=0.16.0" emails pyjwt email_validator jinja2 psycopg2-binary alembic SQLAlchemy email_validator
+ADD requirements.txt .
+RUN pip install -r requirements.txt
```

Comment:
The same here, I would prefer to add Poetry to the Docker images and install with Poetry without creating an env inside the Dockerfiles instead of having a pre-build step outside of Docker that depends on Poetry installed locally.